client/i18n/en.json

```diff
@@ -513,6 +513,14 @@
li>Operative, laboratory and procedure reports.</li></ul><p>Rhinoplasty to repair an external nasal deformity not causing functional impairment <b>is considered cosmetic.</b></p><b>Nasal Valve Suspension</b><p>Nasal valve suspension as a surgical technique for the repair of nasal valve collapse is <b>considered experimental, investigational and/or unproven.</b></p><b><u>Reconstructive Nasal and Sinus Surgery for Congenital or Acquired Functional Nasal Deformities </u></b><p>Nasal surgery performed on the outside or inside of the nose <b>may be considered medically necessary</b> when performed as a reconstructive procedure to correct congenital or acquired functional nasal deformities that meets any one of the specific medical necessity criteria for <b>Rhinoplasty</b> as listed above.</p><b><u>Endoscopic Sinus Surgery, Functional (FESS)</u></b><p>Functional endoscopic sinus surgery <b>may be considered medically necessary</b> when medical management has failed or is unavailable for the following diagnoses:</p><ul><li>Chronic sinusitis refractory to medical treatment,</li><li>Recurrent sinusitis,</li><li>Nasal polyposis,</li><li>Antrochoanal polyps,</li><li>Sinus mucoceles,</li><li>Excision of selected tumors,</li><li>Cerebrospinal fluid (CSF) leak closure,</li><li>Orbital decompression (e.g., Graves ophthalmopathy),</li><li>Optic nerve decompression,</li><li>Dacryocystorhinostomy (DCR),</li><li>Choanal atresia repair,</li><li>Foreign body removal, OR</li><li>Epistaxis control.</li></ul><b><u>Balloon Ostial Dilation</u></b><p>Balloon ostial dilation using a catheter-based inflatable device (e.g. Balloon Sinuplasty™) for the treatment of medically refractory chronic sinusitis <b>may be considered medically necessary</b> as a minimally invasive alternative to functional endoscopic sinus surgery.</p><p><span><strong><u>Implantable Sinus Stents</u></strong></span></p><p>Placement of a mometasone furoate sinus implant (Propel<strong>&trade;) may be considered medically necessary</strong> in conjunction with ethmoid sinus surgery when the following criteria are met:</p><ul><li><span>&bull;</span>Criteria for Functional Endoscopic Sinus Surgery (FESS) noted above are met; AND</li><li><span>&bull;</span>Patient is &gt; 18 years of age.</li></ul><p><span><strong><u>Other Nasal and Sinus Surgical Procedures</u></strong></span></p><p>Procedures that are performed to reshape the normal structures of the nose and improve the appearance, whether performed separately or in combination with another procedure, <strong>are considered cosmetic</strong>&nbsp; These procedures include:</p><ul><li>Changing the size of the nose; or</li><li>Changing the shape of the nose; or</li><li>Narrowing the nostrils; or</li><li>Changing the angle between the nose and lips.</li></ul><p>Cosmetic services that are requested or performed because of psychiatric or emotional problems attributed to the actual or perceived defect being treated <strong>are considered cosmetic</strong>.</p>",
```

Comment:
This solution doesn't seem very eloquent. Is there a way we can set this up to be more flexible. I don't want hardcoded content in this application.